### PR TITLE
Add chrono

### PR DESCRIPTION
**Pull request by wanasit**
https://github.com/wanasit/chrono

### CODE REVIEW

File: files/chrono/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "chrono",
+  "repo": "wanasit/chrono",
+  "files": {
+    "include": ["chrono.js", "chrono.min.js", "chrono.map.json"]
```
Comment (jimaek) [inline files/chrono/update.json]: There is no map.json here https://github.com/wanasit/chrono/tree/v1.0.2
Also I think that just `chrono.min.js` is enough

Comment (wanasit) [inline files/chrono/update.json]: Sorry. Added `chrono.map.json` in v1.1.0.
https://github.com/wanasit/chrono/tree/v1.1.0

Comment (jimaek) [inline files/chrono/update.json]: Is that file needed in production?

Comment (wanasit) [inline files/chrono/update.json]: Well..not really. I just followed the example and it might be useful when debugging the min.js. 
If you suggest put only production needed files in JSDelivr, I will remove it.

Comment (nickpolet) [inline files/chrono/update.json]: I would say that anyone who is looking to debug the file, would have the knowledge to find your github page and look through the un-minified file there. But I am not affiliated with jsdelivr, so maybe they have different views about it.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by bradvogel** [comment]
@wanasit Is this correct? The file isn't working for me:

```
<html>

<!-- This version includes chrono.Refiner -->
<!-- <script src="https://rawgithub.com/wanasit/chrono/master/chrono.min.js"></script> -->

<!-- This version doesn't include chrono.Refiner -->
<script src="https://cdn.jsdelivr.net/chrono/0.1.11/chrono.min.js"></script>
<script>
	if (!chrono.Refiner) alert('chrono.Refiner does not exist');
</script>
<body></body>
</html>
```

**Comment by wanasit** [comment]
It's in jsdelivr. but there is a separate bug in v0.1.11.

Try:
```
<html>
<!-- This version includes chrono.Refiner -->
<!-- <script src="https://rawgithub.com/wanasit/chrono/master/chrono.min.js"></script> -->

<!-- This version doesn't include chrono.Refiner -->
<script src="https://cdn.jsdelivr.net/chrono/0.1.11/chrono.min.js"></script>
<script>
    alert(chrono.parseDate('today at 1 pm');
</script>
<body></body>
</html>
```

**Comment by wanasit** [comment]
Then, the problem is.. why the newer version (1.1.0 and 1.0.2) is not available?

**Comment by bradvogel** [comment]
So `chrono.Refiner` is missing from `v0.1.11`? Then that would make sense. If so, it'd be great to get the most recent (bug free!) version up here.

**Comment by bradvogel** [comment]
ping @wanasit

**Comment by bradvogel** [comment]
@wanasit can you clarify - what's the separate bug? Moving convo to https://github.com/wanasit/chrono/issues/78
